inc/apirest.class.php

```diff
@@ -189,6 +189,13 @@ public function call() {
 
          return $this->returnResponse($response, $code, $additionalheaders);
 
+      } else if ($resource === "lostPassword") {
+         if ($this->verb != 'POST' && $this->verb != 'PUT') {
+            // forbid password reset with GET because it may be logged
+            return $this->returnError();
```

Comment:
OK to send a specific message.

I initially thought GET would be suffficient. I have to update the documentation.